cli/stubs/Caddyfile

```diff
@@ -1,7 +1,7 @@
 import VALET_HOME_PATH/Caddy/*
 
 :80 {
-    fastcgi / /var/run/valet/fpm.socket php {
+    fastcgi / /var/run/valet/fpm-valet.socket php {
```

Comment:
I think you meant to remove the `/valet` subdirectory here, and in the SecureCaddyfile stub too.